### PR TITLE
Adding TagsPage scenario 

### PR DESCRIPTION
**Pull request by yassir-ai**
## Description

What's new?
#1485 

## What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Tests
- [ ] Other

### CODE REVIEW

File: src/AzureIoTHub.Portal.Tests.E2E/DeviceTags.cs
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal.Tests.E2E/DeviceTags.cs:1]: Please rename this class "TagsConfiguration"

File: src/AzureIoTHub.Portal.Tests.E2E/Pages/TagsPage.cs
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal.Tests.E2E/Pages/TagsPage.cs:1]: Please rename this class "TagsConfigurationPage"

File: src/AzureIoTHub.Portal.Tests.E2E/DeviceTags.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        [TearDown]
+        public override void TearDown()
+        {
+            loginPage.Logout();
+
+            base.TearDown();
+        }
+
+        [Test]
+        public void UserCanAddAndRemoveDeviceTag()
```
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal.Tests.E2E/DeviceTags.cs:32]:
```suggestion
        public void UserCanAddAndRemoveTagConfiguration()

```

**Review by kbeaugrand — CHANGES_REQUESTED** [review]

File: src/AzureIoTHub.Portal.Tests.E2E/Pages/TagsPage.cs
Comment (yassir-ai) [reply src/AzureIoTHub.Portal.Tests.E2E/Pages/TagsPage.cs:1]: It's done

File: src/AzureIoTHub.Portal.Tests.E2E/DeviceTags.cs
Comment (Achraf-Git-dev) [reply src/AzureIoTHub.Portal.Tests.E2E/DeviceTags.cs:1]: done

**Review by kbeaugrand — APPROVED** [review]

**Comment by kbeaugrand** [comment]
LGTM : E2E tests did not pass since it's coming from a fork.

**Pull request merged**